python/meterpreter/ext_server_stdapi.py

```diff
@@ -2188,8 +2188,8 @@ def stdapi_net_config_get_interfaces(request, response):
         iface_tlv += tlv_pack(TLV_TYPE_MAC_ADDRESS, iface_info.get('hw_addr', '\x00\x00\x00\x00\x00\x00'))
         if 'mtu' in iface_info:
             iface_tlv += tlv_pack(TLV_TYPE_INTERFACE_MTU, iface_info['mtu'])
-        if 'flags' in iface_info:
-            iface_tlv += tlv_pack(TLV_TYPE_INTERFACE_FLAGS, iface_info['flags'])
+        if 'flags_str' in iface_info:
+            iface_tlv += tlv_pack(TLV_TYPE_INTERFACE_FLAGS, iface_info['flags_str'])
```

Comment:
OSX was returning the integer flag value here, whilst linux was returning the string flag names. Now the implementations return both separately

Reply:
This seems to be introducing new data into the TLV, is there value for both vs just shifting to a more consistent type retuned in `flags`?

Reply:
`TLV_TYPE_INTERFACE_FLAGS` has always been a string:

```
TLV_TYPE_INTERFACE_FLAGS    = TLV_META_TYPE_STRING  | 1403
```

Previously the OSX code path was was extracting out the integer value for flags instead of a string like in the Linux codepath, i.e. in the osx code path attempted to write out `8836` instead of `"UP,BROADCAST,SMART,RUNNING,SIMPLEX,MULTICAST"` from the below ifconfig output

```
en5: flags=8863<UP,BROADCAST,SMART,RUNNING,SIMPLEX,MULTICAST> mtu 1500
```

This is now updated to send the string flag names for both OSX and Linux

Reply:
Got it, `stdapi_net_config_get_interfaces_via_netlink` was returning `flags` as a set of strings names and  `stdapi_net_config_get_interfaces_via_osx_ifconfig` was returning `flags` as numeric values now

Both return `flags` as numeric values and `flags_str` as string names and only `flags_str` is being placed in the response as a `TLV_TYPE_INTERFACE_FLAGS` packet.